src/django/api/tests.py

```diff
@@ -1393,6 +1393,47 @@ def test_approve_facility_claim(self):
 
         self.assertEqual(400, error_response.status_code)
 
+    @override_switch('claim_a_facility', active=True)
+    def test_can_approve_at_most_one_facility_claim(self):
+        self.client.post(
```

Comment:
For the sake of completeness, I will also add a check that the response status from this first request is 200.

Reply:
Done.